packages/moonstone/Scroller/Scroller.js

```diff
@@ -34,84 +34,36 @@ const dataContainerDisabledAttribute = 'data-spotlight-container-disabled';
  * [SpotlightContainerDecorator]{@link spotlight/SpotlightContainerDecorator.SpotlightContainerDecorator}
  * and the Scrollable version, [Scroller]{@link moonstone/Scroller.Scroller}.
  *
- * @class ScrollerBase
+ * @function ScrollerBase
  * @memberof moonstone/Scroller
  * @extends ui/Scroller.ScrollerBase
  * @ui
  * @public
  */
-class ScrollerBase extends Component {
-	static displayName = 'ScrollerBase'
-
-	static propTypes = /** @lends moonstone/Scroller.ScrollerBase.prototype */ {
-		/**
-		 * Passes the instance of [Scroller]{@link ui/Scroller.Scroller}.
-		 *
-		 * @type {Object}
-		 * @param {Object} ref
-		 * @private
-		 */
-		initUiChildRef: PropTypes.func,
-
-		/**
-		 * Called when [Scroller]{@link moonstone/Scroller.Scroller} updates.
-		 *
-		 * @type {function}
-		 * @private
-		 */
-		onUpdate: PropTypes.func,
-
-		/**
-		 * `true` if rtl, `false` if ltr.
-		 *
-		 * @type {Boolean}
-		 * @private
-		 */
-		rtl: PropTypes.bool,
-
-		/**
-		 * Called when [Scroller]{@link moonstone/Scroller.Scroller} should be scrolled
-		 * and the focus should be moved to a scrollbar button.
-		 *
-		 * @type {function}
-		 * @private
-		 */
-		scrollAndFocusScrollbarButton: PropTypes.func,
-
-		/**
-		 * The spotlight id for the component.
-		 *
-		 * @type {String}
-		 * @private
-		 */
-		spotlightId: PropTypes.string
-	}
-
-	componentDidMount () {
-		this.configureSpotlight();
-	}
-
-	componentDidUpdate (prevProps) {
-		const {onUpdate} = this.props;
+const ScrollerBase = (props) => {
+	useEffect(() => {
+		// componentDidMount
+		configureSpotlight();
+
+		// componentWillUnmount
+		return () => setContainerDisabled(false);
+	}, [configureSpotlight, setContainerDisabled]);	// TODO : Handle exhaustive-deps ESLint rule.
+
+	useEffect(configureSpotlight, [props.spotlightId]);	// TODO : Handle exhaustive-deps ESLint rule.
+	useEffect(() => {
+		const {onUpdate} = props;
 		if (onUpdate) {
-			onUpdate();
+		//	onUpdate();		// TODO: Invoking onUpdate() has error. Fix it.
```

Comment:
Do we fix it later? If yes, please write some comments in the PR description and create an issue.

Reply:
@jeonghee27 wrote the following comment in the PR description.
```
Now, some error is occurring on some part to access class instance method using ref.
Function component doesn't have instance, so I will solve the problem on PLAT-98204.
That means, current operation accompany some errors yet.
```
So we'll fix it in another separate issue.

Reply:
I added Issue number at that code